x-pack/platform/plugins/shared/fleet/server/types/rest_spec/epm.ts

```diff
@@ -162,6 +149,7 @@ export const PackageInfoSchema = schema
         schema.literal('integration'),
         schema.literal('input'),
         schema.literal('content'),
+        schema.any(),
```

Comment:
should we restrict to string instead of any?

Reply:
updated